### DISCUSSION TITLE
Бага при установке в htaccess

### DISCUSSION THREAD

**Discussion by GPolyakov** [maxsite/cms#466] [General]
Добрый день.
Устанавливал MaxSite CMS через update-maxsite.
Все сказалось без проблем и установилось в принципе тоже.
Но после установке не смог поспасть в админку и получал ошибку
```No input file specified.```

Решение нашел в интернете
в 18 строку файла ```.htaccess```
необходимо добавить ? после index.php
Итоговая строка должна выглядеть так ```RewriteRule ^(.*)$ /index.php?/$1 [L,QSA] ```
После этого админка запускается

**Comment by GPolyakov** [comment]
надо было сразу Issue заводить, а я чет промахнулся и сюда написал
https://github.com/maxsite/cms/issues/467

**Comment by maxsite** [comment]
Скорее всего какие-то особенные настройки вашего сервера. Возможно он настроен на CGI.

**Reply by GPolyakov** [reply]
Думал что это какая о общая проблема, так как нашёл похожее в интернета. Вроде как проблема не единичная.
Если подскажите какую инфу с сервка вытащить я её прикреплю

**Comment by maxsite** [comment]
Если это не обычный апач, то админы могут настраивать сервер так, что ЧПУ в базовом варианте в .htaccess не будет работать. В целом это достаточно редкие случаи. Как правило типового .htaccess хватает с головой.

**Reply by GPolyakov** [reply]
Да согласен, что тут от настроек зависит.
Но может быть в фак запихнуть что есть такая трабла, решать так то

**Comment by maxsite** [comment]
Так так там есть пример для CGI. Ваш вариант с добавлением `?` мне кажется не совсем корректным, поскольку это перебивает get-параметры, если они будут в URL. Лучше такие вещи уточнять у админа сервера.
